packages/manager/modules/telecom-spare/src/modems/order/modems-order.html

```diff
@@ -0,0 +1,210 @@
+<div class="telecom-spare-modems-order-section">
+    <header>
+        <tuc-section-back-link data-tuc-section-back-link-to-state="spare.modems"
+            data-tuc-section-back-link-title="{{ ::'modems_order_back_link' | translate }}">
+        </tuc-section-back-link>
+        <h1 data-translate="modems_order_new_modem_title"></h1>
+    </header>
+
+    <div class="modems-order-section-content">
+
+        <tuc-toast-message></tuc-toast-message>
+
+        <div class="row">
+            <div class="col">
+                <!-- ORDER -->
+                <section class="widget-presentation" data-ng-if="$ctrl.order.orderURL">
+                    <h2 class="widget-presentation-title" data-translate="modems_order_shipping_title"></h2>
+                    <div class="alert alert-success" role="alert" data-translate="modem_order_order_success"></div>
```

Comment:
Use a `oui-message` instead

Reply:
Sorry but the last time I tried to integrate the oui-message, it didn't work as well I expected.

Reply:
This should be fine :
```suggestion
                    <oui-message data-type="success">
                        <span data-translate="modem_order_order_success">
                    </oui-message>
```